### PR TITLE
Feature/julia source

### PR DESCRIPTION
**Pull request by JHartsek**
Changes Made:
• Created source component 
• Created corresponding story

### CODE REVIEW

File: src/Components/Source.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  color: ${(props) => props.theme.colors.alt_text_white};
+  width: 290px;
+  box-sizing: border-box;
+  border-radius: 8px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 12px;
+  position: absolute;
+  top: 290px;
```
Comment (Samm96) [inline src/Components/Source.js]: I'm wondering if this should be kept to `0` and `position:absolute` shouldn't be applied here so the component can be reused for many other components. And then if its needed to be placed in a certain spot, that can be adjusted in the component this component will be placed in.  i.e.: 

Screen Shot 2022-11-07 at 3 06 16 PM

And then add the `position: absolute` and `top: 290px` to `<AnotherComponent__container>` CSS.

Comment (JHartsek) [reply src/Components/Source.js]: I think that's a good idea. I forgot that other components besides the _Stat_ component had the info button/source popup.

**Comment by pstaal** [comment]
Hi Julia, im am not sure about the info prop:

`const Source = ({ info }) => {
  return (
    <Container>
      <Info>{info}</Info>
    </Container>
  );
};`

This component always renders the i in this page, so I think you can just do this:

`const Source = () => {
  return (
    <Container>
      <Info>&#9432;</Info>
    </Container>
  );
};`

This html code renders the circled i.

**Comment by JHartsek** [comment]
Hi @pstaal , this code was intended to render the cited source popup which might be vary.
image

**Comment by pstaal** [comment]
You are right sorry was mistaken

**Pull request merged**